build.gradle

```
@@ -179,6 +179,30 @@ tasks {
 
 patchPluginXml {
     changeNotes = """
+        <h2> 24.0.3 </h2>
+        Version 24.0.3 of Liberty Tools for IntelliJ IDEA contains enhancements and fixes. Version 24.0.3 requires IntelliJ IDEA version 2023.3.* and requires Java 17 or later.
```

As discussed, we will support 2023.2.x and 2023.3.x.
I suggest we go with the phrasing of the 23.0.12 release notes on GitHub: https://github.com/OpenLiberty/liberty-tools-intellij/releases/tag/23.0.12

```suggestion
        Version 24.0.3 of Liberty Tools for IntelliJ IDEA contains enhancements and fixes. Version 24.0.3 requires IntelliJ IDEA version 2023.2.* or 2023.3.* and a minimum of Java 17.
```

---

Added